viz/vizkit_plugin.rb

```diff
@@ -23,6 +23,22 @@
 # OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #
-Vizkit::UiLoader.register_3d_plugin('SpatioTemporalMLSMapKalman', 'envire_maps', 'SpatioTemporalMLSMapKalman')
-Vizkit::UiLoader.register_3d_plugin_for('SpatioTemporalMLSMapKalman', "/envire/core/SpatioTemporal<maps/grid/MLSMapKalman>", :updateData )
+Vizkit::UiLoader.register_3d_plugin('SpatioTemporalContourMapVisualization', 'envire_maps', 'SpatioTemporalContourMapVisualization')
+Vizkit::UiLoader.register_3d_plugin_for('SpatioTemporalContourMapVisualization', "/envire/core/SpatioTemporal<maps/geometric/ContourMap>", :updateSpatioTemporalContourMap)
```

Comment:
There are '/' missing before the maps namespace. It appears otherwise ruby can't identify the type.
So it should be /envire/core/SpatioTemporal</maps/MapType> for all types here.